modules/visual_script/visual_script_nodes.cpp

```diff
@@ -842,7 +842,8 @@ PropertyInfo VisualScriptOperator::get_input_value_port_info(int p_idx) const {
 		{ Variant::BOOL, Variant::BOOL }, //OP_XOR,
 		{ Variant::BOOL, Variant::BOOL }, //OP_NOT,
 		//containment
-		{ Variant::NIL, Variant::NIL } //OP_IN,
+		{ Variant::NIL, Variant::NIL }, //OP_IN,
```

Comment:
Not sure if this is really needed in Visual Script. In any case, without the operation in Variant this would need more work so it might be best to be done as another Pull Request.

Reply:
As above, not needed anymore, if we don't add a new op to Variant. Adding a NOT-IN Visual Script node is probably be a bit more complicated without the corresponding Variant op.